app/controllers/pensions/shortServiceRefunds/ShortServicePensionsSchemeController.scala

```diff
@@ -93,7 +93,11 @@ class ShortServicePensionsSchemeController @Inject()(actionsProvider: ActionsPro
       schemeReference = (if (scheme.ukRefundCharge.contains(true)) scheme.pstr else scheme.qops)
         .getOrElse(""),
       providerAddress = scheme.providerAddress.getOrElse(""),
-      countryId = scheme.alphaTwoCountryCode
+      countryId = scheme.alphaTwoCountryCode.fold{
```

Comment:
why is this change in here ?

Reply:
was brought up in the group review, for edge cases